src/javascript/app/common/content_visibility.js

```diff
@@ -112,31 +118,25 @@ const ContentVisibility = (() => {
 
         const rule_set_has_current = rule_set.has(current_landing_company_shortcode);
         const rule_set_has_mt      = rule_set.has(mt_company_rule);
+        const rule_set_has_country = rule_set.has(clients_country);
 
         let show_element = false;
 
-        if (client_has_mt_company && rule_set_has_mt) show_element = !is_exclude;
-        else if (is_exclude !== rule_set_has_current) show_element = true;
+        if (client_is_logged_in && client_has_mt_company && rule_set_has_mt) show_element = !is_exclude;
+        else if (client_is_logged_in && is_exclude !== rule_set_has_current) show_element = true;
+        else if (!client_is_logged_in && is_exclude !== rule_set_has_country) show_element = true;
```

Comment:
seems like 126 and 127 are the same, can we just remove the `client_is_logged_in` and `! client_is_logged_in` check to combine them into one?